### ISSUE TITLE
Cannot sign in - Error: Unable to parse decrypted password

### ISSUE THREAD

**Issue by SmithPlatts** [microsoft/vscode#165115]
Type: <b>Bug</b>

When attempting to sign in with my Microsoft account, after auth completes, VSCode requests to sign in again - the following is shown in the Microsoft Authentication Log:

```
[2022-11-01 14:00:11.534] [Microsoft Authentication] [info] Getting sessions for all scopes...
[2022-11-01 14:00:11.534] [Microsoft Authentication] [info] Got 0 sessions for all scopes...
[2022-11-01 14:00:12.479] [Microsoft Authentication] [info] Getting sessions for the following scopes: email offline_access openid profile
[2022-11-01 14:00:12.479] [Microsoft Authentication] [info] Got 0 sessions for scopes: email offline_access openid profile
[2022-11-01 14:00:13.984] [Microsoft Authentication] [info] Logging in for the following scopes: email offline_access openid profile
[2022-11-01 14:00:17.687] [Microsoft Authentication] [info] Exchanging login code for token for scopes: email offline_access openid profile
[2022-11-01 14:00:18.068] [Microsoft Authentication] [info] Exchanging login code for token (for scopes: email offline_access openid profile) succeeded!
[2022-11-01 14:00:18.068] [Microsoft Authentication] [info] Setting token for scopes: email offline_access openid profile
[2022-11-01 14:00:18.068] [Microsoft Authentication] [info] Login successful for scopes: email offline_access openid profile
[2022-11-01 14:00:18.069] [Microsoft Authentication] [info] Token available from cache (for scopes email offline_access openid profile), expires in 5031000 milliseconds
[2022-11-01 14:00:18.071] [Microsoft Authentication] [info] Getting sessions for the following scopes: email offline_access openid profile
[2022-11-01 14:00:18.071] [Microsoft Authentication] [info] Got 1 sessions for scopes: email offline_access openid profile
[2022-11-01 14:00:18.071] [Microsoft Authentication] [info] Token available from cache (for scopes email offline_access openid profile), expires in 5030997 milliseconds
[2022-11-01 14:00:18.071] [Microsoft Authentication] [info] Getting sessions for the following scopes: email offline_access openid profile
[2022-11-01 14:00:18.071] [Microsoft Authentication] [info] Got 1 sessions for scopes: email offline_access openid profile
[2022-11-01 14:00:18.071] [Microsoft Authentication] [info] Token available from cache (for scopes email offline_access openid profile), expires in 5030997 milliseconds
[2022-11-01 14:00:18.897] [Microsoft Authentication] [error] Error: Unable to parse decrypted password
    at y.$getPassword (vscode-file://vscode-app/c:/Users/(username)/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1727:50142)
[2022-11-01 14:00:18.926] [Microsoft Authentication] [info] Session removed in another window with scopes: email offline_access openid profile
[2022-11-01 14:00:18.926] [Microsoft Authentication] [info] Logging out of session 'f8cdef31-a31e-4b4a-93e4-5f571e91255a/1:live.com:00037FFEA9CBEFA3/b5b7e2f0-a480-47c1-bc41-b573ba1e1758'
[2022-11-01 14:00:18.926] [Microsoft Authentication] [info] Sending change event for session that was removed with scopes: email offline_access openid profile
[2022-11-01 14:00:18.926] [Microsoft Authentication] [info] Logged out of session 'f8cdef31-a31e-4b4a-93e4-5f571e91255a/1:live.com:00037FFEA9CBEFA3/b5b7e2f0-a480-47c1-bc41-b573ba1e1758' with scopes: email offline_access openid profile
```

Not sure what else to add.

VS Code version: Code 1.72.2 (d045a5eda657f4d7b676dedbfa7aab8207f8a075, 2022-10-12T22:15:18.074Z)
OS version: Windows_NT x64 10.0.19044
Modes:
Sandboxed: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Xeon(R) W-2235 CPU @ 3.80GHz (12 x 3792)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|undefined|
|Memory (System)|31.73GB (5.93GB free)|
|Process Argv|--crash-reporter-id 78b31145-6d3c-4704-b7f4-5a189d012012|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (32)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-openapi|42C|4.15.8
vscode-caniuse|aka|0.5.4
Bookmarks|ale|13.3.1
xml|Dot|2.5.1
EditorConfig|Edi|0.16.4
windows-explorer-context-menu|ele|3.2.1
vscode-guid|hea|1.5.0
rest-client|hum|0.25.1
formate|Mik|1.2.1
vscode-azurefunctions|ms-|1.8.3
vscode-azureresourcegroups|ms-|0.5.6
csharp|ms-|1.25.0
vscode-dotnet-runtime|ms-|1.6.0
data-workspace-vscode|ms-|0.3.0
mssql|ms-|1.16.0
sql-bindings-vscode|ms-|0.3.0
sql-database-projects-vscode|ms-|0.19.0
remote-containers|ms-|0.255.4
remote-ssh|ms-|0.90.1
remote-ssh-edit|ms-|0.84.0
remote-wsl|ms-|0.72.0
vscode-remote-extensionpack|ms-|0.21.0
azure-account|ms-|0.11.2
powershell|ms-|2022.10.0
azurerm-vscode-tools|msa|0.15.8
autodocstring|njp|0.6.1
vscode-yaml|red|1.10.1
code-spell-checker|str|2.10.1
keyoti-changeallendoflinesequence|vs-|0.0.3
vscode-todo-highlight|way|1.0.5
save-as-root|yy0|1.5.0
markdown-pdf|yza|1.4.4

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
pythonvspyl392:30443607
vserr242cf:30382550
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263cf:30335440
vscorecescf:30445987
pythondataviewer:30285071
vscod805cf:30301675
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593cf:30376535
pythonvs932:30410667
cppdebug:30492333
vsclangdc:30486549
c4g48928:30535728
dsvsc012cf:30540253
azure-dev_surveyone:30548225
pyindex848:30577860
nodejswelcome1cf:30587006
fc301958:30595537
2e4cg342:30596373

```

</details>

**Comment by TylerLeonhardt** [comment]
1.73 was just released. Can you try the update and let me know how it goes?

**Comment by SmithPlatts** [comment]
Updated and retried:

```
2022-11-03 18:11:36.977 [info] Reading sessions from secret storage...
2022-11-03 18:11:44.637 [error] Key was not found in SecretStorage.
2022-11-03 18:11:44.832 [info] Got 0 stored sessions
2022-11-03 18:11:45.765 [info] Getting sessions for the following scopes: email offline_access openid profile
2022-11-03 18:11:45.765 [info] Got 0 sessions for scopes: email offline_access openid profile
2022-11-03 18:12:04.496 [info] Getting sessions for all scopes...
2022-11-03 18:12:04.551 [info] Got 0 sessions for all scopes...
2022-11-03 18:12:05.576 [info] Getting sessions for the following scopes: email offline_access openid profile
2022-11-03 18:12:05.576 [info] Got 0 sessions for scopes: email offline_access openid profile
2022-11-03 18:12:07.085 [info] Logging in for the following scopes: email offline_access openid profile
2022-11-03 18:12:10.894 [info] Exchanging login code for token for scopes: email offline_access openid profile
2022-11-03 18:12:11.373 [info] Exchanging login code for token (for scopes: email offline_access openid profile) succeeded!
2022-11-03 18:12:11.373 [info] Setting token for scopes: email offline_access openid profile
2022-11-03 18:12:11.374 [info] Login successful for scopes: email offline_access openid profile
2022-11-03 18:12:11.374 [info] Token available from cache (for scopes email offline_access openid profile), expires in 5388999 milliseconds
2022-11-03 18:12:11.377 [info] Getting sessions for the following scopes: email offline_access openid profile
2022-11-03 18:12:11.377 [info] Got 1 sessions for scopes: email offline_access openid profile
2022-11-03 18:12:11.377 [info] Token available from cache (for scopes email offline_access openid profile), expires in 5388996 milliseconds
2022-11-03 18:12:11.377 [info] Getting sessions for the following scopes: email offline_access openid profile
2022-11-03 18:12:11.377 [info] Got 1 sessions for scopes: email offline_access openid profile
2022-11-03 18:12:11.377 [info] Token available from cache (for scopes email offline_access openid profile), expires in 5388996 milliseconds
2022-11-03 18:12:12.227 [error] Error: Decryption failed.
2022-11-03 18:12:12.556 [error] Error: Decryption failed.
2022-11-03 18:12:12.556 [error] Error: Decryption failed.
2022-11-03 18:12:12.561 [error] Error: Decryption failed.
2022-11-03 18:12:12.561 [error] Error: Decryption failed.
2022-11-03 18:12:12.566 [error] Error: Decryption failed.
2022-11-03 18:12:12.566 [error] Error: Decryption failed.
2022-11-03 18:12:12.571 [error] Error: Decryption failed.
2022-11-03 18:12:12.623 [error] Error: Decryption failed.
2022-11-03 18:12:12.643 [info] Getting sessions for the following scopes: email offline_access openid profile
2022-11-03 18:12:12.643 [info] Got 1 sessions for scopes: email offline_access openid profile
2022-11-03 18:12:12.643 [info] Token available from cache (for scopes email offline_access openid profile), expires in 5387730 milliseconds
2022-11-03 18:12:14.590 [info] Session removed in another window with scopes: email offline_access openid profile
2022-11-03 18:12:14.590 [info] Logging out of session 'f8cdef31-a31e-4b4a-93e4-5f571e91255a/1:live.com:00037FFEA9CBEFA3/3f8b83ea-c152-48bb-80b3-ab34d2f43447'
2022-11-03 18:12:14.590 [info] Sending change event for session that was removed with scopes: email offline_access openid profile
2022-11-03 18:12:14.590 [info] Logged out of session 'f8cdef31-a31e-4b4a-93e4-5f571e91255a/1:live.com:00037FFEA9CBEFA3/3f8b83ea-c152-48bb-80b3-ab34d2f43447' with scopes: email offline_access openid profile
2022-11-03 18:12:14.602 [info] Getting sessions for the following scopes: email offline_access openid profile
2022-11-03 18:12:14.602 [info] Got 0 sessions for scopes: email offline_access openid profile
2022-11-03 18:12:14.633 [error] Error: Unable to parse decrypted password
    at I.$getPassword (vscode-file://vscode-app/c:/Users/${USER}/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1736:49238)
2022-11-03 18:12:14.633 [error] Error: Unable to parse decrypted password
    at I.$getPassword (vscode-file://vscode-app/c:/Users/${USER}/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1736:49238)
2022-11-03 18:12:14.633 [error] Error: Unable to parse decrypted password
    at I.$getPassword (vscode-file://vscode-app/c:/Users/${USER}/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1736:49238)
2022-11-03 18:12:14.633 [error] Error: Unable to parse decrypted password
    at I.$getPassword (vscode-file://vscode-app/c:/Users/${USER}/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1736:49238)
2022-11-03 18:12:14.633 [error] Error: Unable to parse decrypted password
    at I.$getPassword (vscode-file://vscode-app/c:/Users/${USER}/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1736:49238)
2022-11-03 18:12:14.633 [error] Error: Unable to parse decrypted password
    at I.$getPassword (vscode-file://vscode-app/c:/Users/${USER}/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1736:49238)
2022-11-03 18:12:14.633 [error] Error: Unable to parse decrypted password
    at I.$getPassword (vscode-file://vscode-app/c:/Users/${USER}/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1736:49238)
2022-11-03 18:12:14.633 [error] Error: Unable to parse decrypted password
    at I.$getPassword (vscode-file://vscode-app/c:/Users/${USER}/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1736:49238)
2022-11-03 18:12:14.633 [error] Error: Unable to parse decrypted password
    at I.$getPassword (vscode-file://vscode-app/c:/Users/${USER}/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1736:49238)
2022-11-03 18:12:14.633 [error] Error: Unable to parse decrypted password
    at I.$getPassword (vscode-file://vscode-app/c:/Users/${USER}/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1736:49238)
2022-11-03 18:12:14.633 [error] Error: Unable to parse decrypted password
    at I.$getPassword (vscode-file://vscode-app/c:/Users/${USER}/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1736:49238)
2022-11-03 18:12:14.633 [error] Error: Unable to parse decrypted password
    at I.$getPassword (vscode-file://vscode-app/c:/Users/${USER}/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1736:49238)
2022-11-03 18:12:14.633 [error] Error: Unable to parse decrypted password
    at I.$getPassword (vscode-file://vscode-app/c:/Users/${USER}/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1736:49238)
2022-11-03 18:12:14.634 [error] Error: Unable to parse decrypted password
    at I.$getPassword (vscode-file://vscode-app/c:/Users/${USER}/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1736:49238)
2022-11-03 18:12:14.634 [error] Error: Unable to parse decrypted password
    at I.$getPassword (vscode-file://vscode-app/c:/Users/${USER}/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1736:49238)
2022-11-03 18:12:14.634 [error] Error: Unable to parse decrypted password
    at I.$getPassword (vscode-file://vscode-app/c:/Users/${USER}/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1736:49238)
2022-11-03 18:12:14.634 [error] Error: Unable to parse decrypted password
    at I.$getPassword (vscode-file://vscode-app/c:/Users/${USER}/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1736:49238)
2022-11-03 18:12:14.634 [error] Error: Unable to parse decrypted password
    at I.$getPassword (vscode-file://vscode-app/c:/Users/${USER}/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1736:49238)
2022-11-03 18:12:14.634 [error] Error: Unable to parse decrypted password
    at I.$getPassword (vscode-file://vscode-app/c:/Users/${USER}/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1736:49238)
2022-11-03 18:12:14.634 [error] Error: Unable to parse decrypted password
    at I.$getPassword (vscode-file://vscode-app/c:/Users/${USER}/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1736:49238)
2022-11-03 18:12:14.634 [error] Error: Unable to parse decrypted password
    at I.$getPassword (vscode-file://vscode-app/c:/Users/${USER}/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1736:49238)
2022-11-03 18:12:14.634 [error] Error: Unable to parse decrypted password
    at I.$getPassword (vscode-file://vscode-app/c:/Users/${USER}/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1736:49238)
2022-11-03 18:12:14.659 [error] Error: Unable to parse decrypted password
    at I.$getPassword (vscode-file://vscode-app/c:/Users/${USER}/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1736:49238)
2022-11-03 18:12:14.661 [error] Error: Unable to parse decrypted password
    at I.$getPassword (vscode-file://vscode-app/c:/Users/${USER}/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1736:49238)
2022-11-03 18:12:14.661 [error] Error: Unable to parse decrypted password
    at I.$getPassword (vscode-file://vscode-app/c:/Users/${USER}/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1736:49238)
2022-11-03 18:12:14.661 [error] Error: Unable to parse decrypted password
    at I.$getPassword (vscode-file://vscode-app/c:/Users/${USER}/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1736:49238)
2022-11-03 18:12:14.662 [error] Error: Unable to parse decrypted password
    at I.$getPassword (vscode-file://vscode-app/c:/Users/${USER}/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1736:49238)
2022-11-03 18:12:14.662 [error] Error: Unable to parse decrypted password
    at I.$getPassword (vscode-file://vscode-app/c:/Users/${USER}/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1736:49238)
2022-11-03 18:12:14.662 [error] Error: Unable to parse decrypted password
    at I.$getPassword (vscode-file://vscode-app/c:/Users/${USER}/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1736:49238)
2022-11-03 18:12:14.662 [error] Error: Unable to parse decrypted password
    at I.$getPassword (vscode-file://vscode-app/c:/Users/${USER}/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1736:49238)
2022-11-03 18:12:14.662 [error] Error: Unable to parse decrypted password
    at I.$getPassword (vscode-file://vscode-app/c:/Users/${USER}/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1736:49238)
2022-11-03 18:12:14.662 [error] Error: Unable to parse decrypted password
    at I.$getPassword (vscode-file://vscode-app/c:/Users/${USER}/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1736:49238)
2022-11-03 18:12:14.662 [error] Error: Unable to parse decrypted password
    at I.$getPassword (vscode-file://vscode-app/c:/Users/${USER}/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1736:49238)
2022-11-03 18:12:14.662 [error] Error: Unable to parse decrypted password
    at I.$getPassword (vscode-file://vscode-app/c:/Users/${USER}/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1736:49238)
2022-11-03 18:12:14.662 [error] Error: Unable to parse decrypted password
    at I.$getPassword (vscode-file://vscode-app/c:/Users/${USER}/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1736:49238)
```

**Comment by TylerLeonhardt** [comment]
@SmithPlatts just to see if we can get you unblocked, can you try this troubleshooting step:
https://code.visualstudio.com/docs/editor/settings-sync#_windows

**Comment by SmithPlatts** [comment]
Hey @TylerLeonhardt 

I have confirmed this behaviour is happening on two different machines, and after clearing the cred cache (17 entries):

```
2022-11-04 23:06:17.793 [info] Reading sessions from secret storage...
2022-11-04 23:06:17.793 [info] Got 0 stored sessions
2022-11-04 23:06:17.793 [info] Getting sessions for the following scopes: email offline_access openid profile
2022-11-04 23:06:17.793 [info] Got 0 sessions for scopes: email offline_access openid profile
2022-11-04 23:06:22.126 [info] Getting sessions for all scopes...
2022-11-04 23:06:22.126 [info] Got 0 sessions for all scopes...
2022-11-04 23:06:23.718 [info] Getting sessions for the following scopes: email offline_access openid profile
2022-11-04 23:06:23.718 [info] Got 0 sessions for scopes: email offline_access openid profile
2022-11-04 23:06:26.148 [info] Logging in for the following scopes: email offline_access openid profile
2022-11-04 23:06:30.257 [info] Exchanging login code for token for scopes: email offline_access openid profile
2022-11-04 23:06:30.508 [info] Exchanging login code for token (for scopes: email offline_access openid profile) succeeded!
2022-11-04 23:06:30.508 [info] Setting token for scopes: email offline_access openid profile
2022-11-04 23:06:30.509 [info] Login successful for scopes: email offline_access openid profile
2022-11-04 23:06:30.509 [info] Token available from cache (for scopes email offline_access openid profile), expires in 4761998 milliseconds
2022-11-04 23:06:30.516 [info] Getting sessions for the following scopes: email offline_access openid profile
2022-11-04 23:06:30.516 [info] Got 1 sessions for scopes: email offline_access openid profile
2022-11-04 23:06:30.516 [info] Token available from cache (for scopes email offline_access openid profile), expires in 4761991 milliseconds
2022-11-04 23:06:30.517 [info] Getting sessions for the following scopes: email offline_access openid profile
2022-11-04 23:06:30.517 [info] Got 1 sessions for scopes: email offline_access openid profile
2022-11-04 23:06:30.517 [info] Token available from cache (for scopes email offline_access openid profile), expires in 4761991 milliseconds
2022-11-04 23:06:31.078 [error] Error: Unable to transform input.
2022-11-04 23:06:31.104 [info] Session removed in another window with scopes: email offline_access openid profile
2022-11-04 23:06:31.104 [info] Logging out of session 'f8cdef31-a31e-4b4a-93e4-5f571e91255a/1:live.com:00037FFEA9CBEFA3/09b20eb7-c22e-41f2-bb26-3b40377de827'
2022-11-04 23:06:31.104 [info] Sending change event for session that was removed with scopes: email offline_access openid profile
2022-11-04 23:06:31.104 [info] Logged out of session 'f8cdef31-a31e-4b4a-93e4-5f571e91255a/1:live.com:00037FFEA9CBEFA3/09b20eb7-c22e-41f2-bb26-3b40377de827' with scopes: email offline_access openid profile
2022-11-04 23:06:31.109 [info] Getting sessions for the following scopes: email offline_access openid profile
2022-11-04 23:06:31.109 [info] Got 0 sessions for scopes: email offline_access openid profile
```

On this machine, I have three windows open at all times when using VSCode.

**Comment by TylerLeonhardt** [comment]
> Error: Unable to transform input.

That's not the same error... do you get this new one on both machines? Has the other error gone away?

Can you run this and tell me what it spits out?:
```
cmdkey /list | Select-String -Pattern "LegacyGeneric:target=(vscode.+)"
```

**Comment by SmithPlatts** [comment]
Sorry, I didn't pick up on the error change! Was head deep in debugging a Durandal and Knockout problem.

So, the laptop, which is where I ran the commends and pasted the latest output, despite multiple attempts last night, decided today that it was going to sign in first try after a fresh launch; output from credman:

```
> cmdkey /list | Select-String -Pattern "LegacyGeneric:target=(vscode.+)"

    Target: LegacyGeneric:target=vscodevscode.microsoft-authentication/microsoft.login.keylist
    Target: LegacyGeneric:target=vscodevscode.microsoft-authentication/f8cdef31-a31e-4b4a-93e4-5f571e91255a/1:live.com:00037FFEA9CBEFA3/7f233caa-f922-433e-8385-fe0f0ec7bfd9-1
    Target: LegacyGeneric:target=vscodevscode.microsoft-authentication/f8cdef31-a31e-4b4a-93e4-5f571e91255a/1:live.com:00037FFEA9CBEFA3/7f233caa-f922-433e-8385-fe0f0ec7bfd9
    Target: LegacyGeneric:target=vscodevscode.microsoft-authentication/f8cdef31-a31e-4b4a-93e4-5f571e91255a/1:live.com:00037FFEA9CBEFA3/09b20eb7-c22e-41f2-bb26-3b40377de827-1
```

I'm about to try on my desktop, which is where I logged this issue from originally.

**Comment by SmithPlatts** [comment]
Ok, desktop:

1. Ran cmdkey list:
```
PS> cmdkey /list | Select-String -Pattern "LegacyGeneric:target=(vscode.+)"

    Target: LegacyGeneric:target=vscodevscode.microsoft-authentication/f8cdef31-a31e-4b4a-93e4-5f571e91255a/1:live.com:00037FFEA9CBEFA3/f0d37cfb-dd2a-4cce-98e0-c7e63d7412c3-1
    Target: LegacyGeneric:target=vscodevscode.microsoft-authentication/microsoft.login.keylist
    Target: LegacyGeneric:target=vscodevscode.microsoft-authentication/f8cdef31-a31e-4b4a-93e4-5f571e91255a/1:live.com:00037FFEA9CBEFA3/f0d37cfb-dd2a-4cce-98e0-c7e63d7412c3-3
    Target: LegacyGeneric:target=vscodevscode.microsoft-authentication/f8cdef31-a31e-4b4a-93e4-5f571e91255a/1:live.com:00037FFEA9CBEFA3/880b9bb9-d03a-41ac-a788-def390bbbf45-2
    Target: LegacyGeneric:target=vscodevscode.microsoft-authentication/f8cdef31-a31e-4b4a-93e4-5f571e91255a/1:live.com:00037FFEA9CBEFA3/bfc45354-2157-4fb3-b207-5f610900e756-1
    Target: LegacyGeneric:target=vscodevscode.microsoft-authentication/f8cdef31-a31e-4b4a-93e4-5f571e91255a/1:live.com:00037FFEA9CBEFA3/70ed9744-9dcc-44dd-912f-dca53f7f246b-1
    Target: LegacyGeneric:target=vscodevscode.microsoft-authentication/f8cdef31-a31e-4b4a-93e4-5f571e91255a/1:live.com:00037FFEA9CBEFA3/b5b7e2f0-a480-47c1-bc41-b573ba1e1758-2
    Target: LegacyGeneric:target=vscodevscode.microsoft-authentication/f8cdef31-a31e-4b4a-93e4-5f571e91255a/1:live.com:00037FFEA9CBEFA3/4fdbda47-c22f-496e-9029-9fde795d5c4f-1
    Target: LegacyGeneric:target=vscodevscode.microsoft-authentication/f8cdef31-a31e-4b4a-93e4-5f571e91255a/1:live.com:00037FFEA9CBEFA3/f0d37cfb-dd2a-4cce-98e0-c7e63d7412c3-2
    Target: LegacyGeneric:target=vscodevscode.microsoft-authentication/f8cdef31-a31e-4b4a-93e4-5f571e91255a/1:live.com:00037FFEA9CBEFA3/b5b7e2f0-a480-47c1-bc41-b573ba1e1758-1
```
2. Started up VSCode (1.73) and attempted sign-in:
```
2022-11-05 14:57:28.981 [info] Reading sessions from secret storage...
2022-11-05 14:57:29.261 [error] Key was not found in SecretStorage.
2022-11-05 14:57:29.261 [info] Got 0 stored sessions
2022-11-05 14:57:29.342 [info] Getting sessions for the following scopes: email offline_access openid profile
2022-11-05 14:57:29.343 [info] Got 0 sessions for scopes: email offline_access openid profile
2022-11-05 14:59:37.883 [info] Getting sessions for all scopes...
2022-11-05 14:59:37.884 [info] Got 0 sessions for all scopes...
2022-11-05 14:59:45.239 [info] Getting sessions for all scopes...
2022-11-05 14:59:45.240 [info] Got 0 sessions for all scopes...
2022-11-05 14:59:46.122 [info] Getting sessions for the following scopes: email offline_access openid profile
2022-11-05 14:59:46.122 [info] Got 0 sessions for scopes: email offline_access openid profile
2022-11-05 14:59:48.497 [info] Logging in for the following scopes: email offline_access openid profile
2022-11-05 14:59:52.578 [info] Exchanging login code for token for scopes: email offline_access openid profile
2022-11-05 14:59:53.157 [info] Exchanging login code for token (for scopes: email offline_access openid profile) succeeded!
2022-11-05 14:59:53.157 [info] Setting token for scopes: email offline_access openid profile
2022-11-05 14:59:53.158 [info] Login successful for scopes: email offline_access openid profile
2022-11-05 14:59:53.158 [info] Token available from cache (for scopes email offline_access openid profile), expires in 4931998 milliseconds
2022-11-05 14:59:53.164 [info] Getting sessions for the following scopes: email offline_access openid profile
2022-11-05 14:59:53.164 [info] Got 1 sessions for scopes: email offline_access openid profile
2022-11-05 14:59:53.164 [info] Token available from cache (for scopes email offline_access openid profile), expires in 4931993 milliseconds
2022-11-05 14:59:53.164 [info] Getting sessions for the following scopes: email offline_access openid profile
2022-11-05 14:59:53.164 [info] Got 1 sessions for scopes: email offline_access openid profile
2022-11-05 14:59:53.164 [info] Token available from cache (for scopes email offline_access openid profile), expires in 4931993 milliseconds
2022-11-05 14:59:53.779 [info] Session removed in another window with scopes: email offline_access openid profile
2022-11-05 14:59:53.779 [info] Logging out of session 'f8cdef31-a31e-4b4a-93e4-5f571e91255a/1:live.com:00037FFEA9CBEFA3/e1ff3c09-636e-4383-bbde-791c7f8eca69'
2022-11-05 14:59:53.779 [info] Sending change event for session that was removed with scopes: email offline_access openid profile
2022-11-05 14:59:53.779 [info] Logged out of session 'f8cdef31-a31e-4b4a-93e4-5f571e91255a/1:live.com:00037FFEA9CBEFA3/e1ff3c09-636e-4383-bbde-791c7f8eca69' with scopes: email offline_access openid profile
2022-11-05 14:59:53.788 [info] Getting sessions for the following scopes: email offline_access openid profile
2022-11-05 14:59:53.788 [info] Got 0 sessions for scopes: email offline_access openid profile
2022-11-05 14:59:53.900 [error] Error: Decryption failed.
2022-11-05 14:59:54.062 [error] Error: Unable to parse decrypted password
    at I.$getPassword (vscode-file://vscode-app/c:/Users/${USER}/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1736:49238)
```
3. Cleared credman cache:
```
PS> cmdkey /list | Select-String -Pattern "LegacyGeneric:target=(vscode.+)" | ForEach-Object { cmdkey.exe /delete $_.Matches.Groups[1].Value }

CMDKEY: Credential deleted successfully.
CMDKEY: Credential deleted successfully.
CMDKEY: Credential deleted successfully.
CMDKEY: Credential deleted successfully.
CMDKEY: Credential deleted successfully.
CMDKEY: Credential deleted successfully.
CMDKEY: Credential deleted successfully.
CMDKEY: Credential deleted successfully.
CMDKEY: Credential deleted successfully.
CMDKEY: Credential deleted successfully.
CMDKEY: Credential deleted successfully.
CMDKEY: Credential deleted successfully.

PS> cmdkey /list | Select-String -Pattern "LegacyGeneric:target=(vscode.+)"
PS>
```
4. Attempted sign-in:
```
2022-11-05 15:02:03.223 [info] Getting sessions for all scopes...
2022-11-05 15:02:03.223 [info] Got 0 sessions for all scopes...
2022-11-05 15:02:06.714 [info] Getting sessions for all scopes...
2022-11-05 15:02:06.714 [info] Got 0 sessions for all scopes...
2022-11-05 15:02:07.971 [info] Getting sessions for the following scopes: email offline_access openid profile
2022-11-05 15:02:07.971 [info] Got 0 sessions for scopes: email offline_access openid profile
2022-11-05 15:02:09.529 [info] Logging in for the following scopes: email offline_access openid profile
2022-11-05 15:02:13.051 [info] Exchanging login code for token for scopes: email offline_access openid profile
2022-11-05 15:02:13.629 [info] Exchanging login code for token (for scopes: email offline_access openid profile) succeeded!
2022-11-05 15:02:13.630 [info] Setting token for scopes: email offline_access openid profile
2022-11-05 15:02:13.630 [info] Login successful for scopes: email offline_access openid profile
2022-11-05 15:02:13.630 [info] Token available from cache (for scopes email offline_access openid profile), expires in 4045999 milliseconds
2022-11-05 15:02:13.635 [info] Getting sessions for the following scopes: email offline_access openid profile
2022-11-05 15:02:13.635 [info] Got 1 sessions for scopes: email offline_access openid profile
2022-11-05 15:02:13.635 [info] Token available from cache (for scopes email offline_access openid profile), expires in 4045994 milliseconds
2022-11-05 15:02:13.635 [info] Getting sessions for the following scopes: email offline_access openid profile
2022-11-05 15:02:13.635 [info] Got 1 sessions for scopes: email offline_access openid profile
2022-11-05 15:02:13.635 [info] Token available from cache (for scopes email offline_access openid profile), expires in 4045994 milliseconds
2022-11-05 15:02:14.254 [error] Error: Unable to transform input.
2022-11-05 15:02:14.261 [info] Session removed in another window with scopes: email offline_access openid profile
2022-11-05 15:02:14.261 [info] Logging out of session 'f8cdef31-a31e-4b4a-93e4-5f571e91255a/1:live.com:00037FFEA9CBEFA3/f008ffd2-1b88-4f38-b46c-36e244a2cfda'
2022-11-05 15:02:14.261 [info] Sending change event for session that was removed with scopes: email offline_access openid profile
2022-11-05 15:02:14.261 [info] Logged out of session 'f8cdef31-a31e-4b4a-93e4-5f571e91255a/1:live.com:00037FFEA9CBEFA3/f008ffd2-1b88-4f38-b46c-36e244a2cfda' with scopes: email offline_access openid profile
2022-11-05 15:02:14.265 [info] Getting sessions for the following scopes: email offline_access openid profile
2022-11-05 15:02:14.265 [info] Got 0 sessions for scopes: email offline_access openid profile
2022-11-05 15:02:14.319 [error] Error: Unable to parse decrypted password
    at I.$getPassword (vscode-file://vscode-app/c:/Users/${USER}/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1736:49238)
2022-11-05 15:02:14.383 [error] Error: Decryption failed.
2022-11-05 15:02:14.505 [error] Error: Decryption failed.
2022-11-05 15:02:14.539 [error] Error: Unable to parse decrypted password
    at I.$getPassword (vscode-file://vscode-app/c:/Users/${USER}/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1736:49238)
2022-11-05 15:02:14.539 [error] Error: Unable to parse decrypted password
    at I.$getPassword (vscode-file://vscode-app/c:/Users/${USER}/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1736:49238)
2022-11-05 15:02:14.539 [error] Error: Unable to parse decrypted password
    at I.$getPassword (vscode-file://vscode-app/c:/Users/${USER}/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1736:49238)
2022-11-05 15:02:14.541 [error] Error: Unable to parse decrypted password
    at I.$getPassword (vscode-file://vscode-app/c:/Users/${USER}/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1736:49238)
2022-11-05 15:02:14.562 [error] Error: Unable to parse decrypted password
    at I.$getPassword (vscode-file://vscode-app/c:/Users/${USER}/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1736:49238)
```
```
PS> cmdkey /list | Select-String -Pattern "LegacyGeneric:target=(vscode.+)"

    Target: LegacyGeneric:target=vscodevscode.microsoft-authentication/microsoft.login.keylist
    Target: LegacyGeneric:target=vscodevscode.microsoft-authentication/f8cdef31-a31e-4b4a-93e4-5f571e91255a/1:live.com:00037FFEA9CBEFA3/f008ffd2-1b88-4f38-b46c-36e244a2cfda-2
    Target: LegacyGeneric:target=vscodevscode.microsoft-authentication/f8cdef31-a31e-4b4a-93e4-5f571e91255a/1:live.com:00037FFEA9CBEFA3/f008ffd2-1b88-4f38-b46c-36e244a2cfda-1
    Target: LegacyGeneric:target=vscodevscode.microsoft-authentication/f8cdef31-a31e-4b4a-93e4-5f571e91255a/1:live.com:00037FFEA9CBEFA3/f008ffd2-1b88-4f38-b46c-36e244a2cfda-3
```
5. Exited from all instances of VSCode (on desktop, I have 5 persistent windows)
6. Launched VSCode and attempted sign-in:
```
2022-11-05 15:06:07.439 [info] Reading sessions from secret storage...
2022-11-05 15:06:09.801 [error] Key was not found in SecretStorage.
2022-11-05 15:06:09.802 [info] Got 0 stored sessions
2022-11-05 15:06:09.829 [info] Getting sessions for the following scopes: email offline_access openid profile
2022-11-05 15:06:09.829 [info] Got 0 sessions for scopes: email offline_access openid profile
2022-11-05 15:06:21.608 [info] Getting sessions for all scopes...
2022-11-05 15:06:21.609 [info] Got 0 sessions for all scopes...
2022-11-05 15:06:23.485 [info] Getting sessions for the following scopes: email offline_access openid profile
2022-11-05 15:06:23.485 [info] Got 0 sessions for scopes: email offline_access openid profile
2022-11-05 15:06:24.911 [info] Logging in for the following scopes: email offline_access openid profile
2022-11-05 15:06:27.871 [info] Exchanging login code for token for scopes: email offline_access openid profile
2022-11-05 15:06:28.323 [info] Exchanging login code for token (for scopes: email offline_access openid profile) succeeded!
2022-11-05 15:06:28.323 [info] Setting token for scopes: email offline_access openid profile
2022-11-05 15:06:28.324 [info] Login successful for scopes: email offline_access openid profile
2022-11-05 15:06:28.325 [info] Token available from cache (for scopes email offline_access openid profile), expires in 5017998 milliseconds
2022-11-05 15:06:28.329 [info] Getting sessions for the following scopes: email offline_access openid profile
2022-11-05 15:06:28.329 [info] Got 1 sessions for scopes: email offline_access openid profile
2022-11-05 15:06:28.329 [info] Token available from cache (for scopes email offline_access openid profile), expires in 5017993 milliseconds
2022-11-05 15:06:28.329 [info] Getting sessions for the following scopes: email offline_access openid profile
2022-11-05 15:06:28.329 [info] Got 1 sessions for scopes: email offline_access openid profile
2022-11-05 15:06:28.329 [info] Token available from cache (for scopes email offline_access openid profile), expires in 5017993 milliseconds
```
```
PS> cmdkey /list | Select-String -Pattern "LegacyGeneric:target=(vscode.+)"

    Target: LegacyGeneric:target=vscodevscode.microsoft-authentication/microsoft.login.keylist
    Target: LegacyGeneric:target=vscodevscode.microsoft-authentication/f8cdef31-a31e-4b4a-93e4-5f571e91255a/1:live.com:00037FFEA9CBEFA3/f008ffd2-1b88-4f38-b46c-36e244a2cfda-2
    Target: LegacyGeneric:target=vscodevscode.microsoft-authentication/f8cdef31-a31e-4b4a-93e4-5f571e91255a/1:live.com:00037FFEA9CBEFA3/dfcd049f-ec95-428d-ab1c-cfeda010e341-1
    Target: LegacyGeneric:target=vscodevscode.microsoft-authentication/f8cdef31-a31e-4b4a-93e4-5f571e91255a/1:live.com:00037FFEA9CBEFA3/f008ffd2-1b88-4f38-b46c-36e244a2cfda-1
    Target: LegacyGeneric:target=vscodevscode.microsoft-authentication/f8cdef31-a31e-4b4a-93e4-5f571e91255a/1:live.com:00037FFEA9CBEFA3/f008ffd2-1b88-4f38-b46c-36e244a2cfda-3
    Target: LegacyGeneric:target=vscodevscode.microsoft-authentication/f8cdef31-a31e-4b4a-93e4-5f571e91255a/1:live.com:00037FFEA9CBEFA3/dfcd049f-ec95-428d-ab1c-cfeda010e341
```

This last time seemed to have worked. All 5 windows report signed-in, and that persisted another exit from all instances and relaunch.

I realise from my initial issue creation, I didn't advise that I *had* performed `cmdkey /list | Select-String -Pattern "LegacyGeneric:target=(vscode.+)" | ForEach-Object { cmdkey.exe /delete $_.Matches.Groups[1].Value }` before logging the issue, but that was on 1.72.2.

**Comment by SmithPlatts** [comment]
I'm back!

Desktop had been off for four days, before this was signed in and happy.

This morning, wouldn't sign-in. CmdKey output:

```
PS> cmdkey /list | Select-String -Pattern "LegacyGeneric:target=(vscode.+)"

    Target: LegacyGeneric:target=vscodevscode.microsoft-authentication/f8cdef31-a31e-4b4a-93e4-5f571e91255a/1:live.com:00037FFEA9CBEFA3/02bc4fe4-6b04-4b6d-80a1-59c3ea97572a
    Target: LegacyGeneric:target=vscodevscode.microsoft-authentication/f8cdef31-a31e-4b4a-93e4-5f571e91255a/1:live.com:00037FFEA9CBEFA3/02bc4fe4-6b04-4b6d-80a1-59c3ea97572a-1
    Target: LegacyGeneric:target=vscodevscode.microsoft-authentication/f8cdef31-a31e-4b4a-93e4-5f571e91255a/1:live.com:00037FFEA9CBEFA3/130c0b51-de07-410e-8ed5-597616d2e08f-1
    Target: LegacyGeneric:target=vscodevscode.microsoft-authentication/microsoft.login.keylist
    Target: LegacyGeneric:target=vscodevscode.microsoft-authentication/f8cdef31-a31e-4b4a-93e4-5f571e91255a/1:live.com:00037FFEA9CBEFA3/f008ffd2-1b88-4f38-b46c-36e244a2cfda-2
    Target: LegacyGeneric:target=vscodevscode.microsoft-authentication/f8cdef31-a31e-4b4a-93e4-5f571e91255a/1:live.com:00037FFEA9CBEFA3/faa5678a-81eb-43f0-ac84-ba9fdcc70d7c-1
    Target: LegacyGeneric:target=vscodevscode.microsoft-authentication/f8cdef31-a31e-4b4a-93e4-5f571e91255a/1:live.com:00037FFEA9CBEFA3/cf816c3b-b90e-44ec-8c22-2e12c52c90d0-2
    Target: LegacyGeneric:target=vscodevscode.microsoft-authentication/f8cdef31-a31e-4b4a-93e4-5f571e91255a/1:live.com:00037FFEA9CBEFA3/130c0b51-de07-410e-8ed5-597616d2e08f-2
    Target: LegacyGeneric:target=vscodevscode.microsoft-authentication/f8cdef31-a31e-4b4a-93e4-5f571e91255a/1:live.com:00037FFEA9CBEFA3/f008ffd2-1b88-4f38-b46c-36e244a2cfda-1
    Target: LegacyGeneric:target=vscodevscode.microsoft-authentication/f8cdef31-a31e-4b4a-93e4-5f571e91255a/1:live.com:00037FFEA9CBEFA3/02bc4fe4-6b04-4b6d-80a1-59c3ea97572a-2
    Target: LegacyGeneric:target=vscodevscode.microsoft-authentication/f8cdef31-a31e-4b4a-93e4-5f571e91255a/1:live.com:00037FFEA9CBEFA3/f008ffd2-1b88-4f38-b46c-36e244a2cfda-3
    Target: LegacyGeneric:target=vscodevscode.microsoft-authentication/f8cdef31-a31e-4b4a-93e4-5f571e91255a/1:live.com:00037FFEA9CBEFA3/dfcd049f-ec95-428d-ab1c-cfeda010e341-4
    Target: LegacyGeneric:target=vscodevscode.microsoft-authentication/f8cdef31-a31e-4b4a-93e4-5f571e91255a/1:live.com:00037FFEA9CBEFA3/130c0b51-de07-410e-8ed5-597616d2e08f-3
```

Although I didn't capture it, the log stated `[error] Error: Unable to parse decrypted password`.

Before I tried signing in, I let VSCode update, and it is now at:

```
Version: 1.73.1 (user setup)
Commit: 6261075646f055b99068d3688932416f2346dd3b
Date: 2022-11-09T04:27:29.066Z
Electron: 19.0.17
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.19045
Sandboxed: No
```

Nuked the CmdKey entries:

```
PS> cmdkey /list | Select-String -Pattern "LegacyGeneric:target=(vscode.+)" | ForEach-Object { cmdkey.exe /delete $_.Matches.Groups[1].Value }

CMDKEY: Credential deleted successfully.
CMDKEY: Credential deleted successfully.
CMDKEY: Credential deleted successfully.
CMDKEY: Credential deleted successfully.
CMDKEY: Credential deleted successfully.
CMDKEY: Credential deleted successfully.
CMDKEY: Credential deleted successfully.
CMDKEY: Credential deleted successfully.
CMDKEY: Credential deleted successfully.
CMDKEY: Credential deleted successfully.
CMDKEY: Credential deleted successfully.
CMDKEY: Credential deleted successfully.
CMDKEY: Credential deleted successfully.
CMDKEY: Credential deleted successfully.
CMDKEY: Credential deleted successfully.
CMDKEY: Credential deleted successfully.
CMDKEY: Credential deleted successfully.
```

Launched VSCode, opened log, and attempted sign-in:

```
2022-11-15 09:45:01.923 [info] Reading sessions from secret storage...
2022-11-15 09:45:05.338 [info] Got 0 stored sessions
2022-11-15 09:45:05.510 [info] Getting sessions for the following scopes: email offline_access openid profile
2022-11-15 09:45:05.510 [info] Got 0 sessions for scopes: email offline_access openid profile
2022-11-15 09:45:33.820 [info] Getting sessions for all scopes...
2022-11-15 09:45:33.821 [info] Got 0 sessions for all scopes...
2022-11-15 09:45:34.692 [info] Getting sessions for the following scopes: email offline_access openid profile
2022-11-15 09:45:34.692 [info] Got 0 sessions for scopes: email offline_access openid profile
2022-11-15 09:45:36.881 [info] Logging in for the following scopes: email offline_access openid profile
2022-11-15 09:45:39.766 [info] Exchanging login code for token for scopes: email offline_access openid profile
2022-11-15 09:45:39.971 [info] Exchanging login code for token (for scopes: email offline_access openid profile) succeeded!
2022-11-15 09:45:39.971 [info] Setting token for scopes: email offline_access openid profile
2022-11-15 09:45:39.972 [info] Login successful for scopes: email offline_access openid profile
2022-11-15 09:45:39.972 [info] Token available from cache (for scopes email offline_access openid profile), expires in 4206999 milliseconds
2022-11-15 09:45:39.978 [info] Getting sessions for the following scopes: email offline_access openid profile
2022-11-15 09:45:39.978 [info] Got 1 sessions for scopes: email offline_access openid profile
2022-11-15 09:45:39.979 [info] Token available from cache (for scopes email offline_access openid profile), expires in 4206993 milliseconds
2022-11-15 09:45:39.979 [info] Getting sessions for the following scopes: email offline_access openid profile
2022-11-15 09:45:39.979 [info] Got 1 sessions for scopes: email offline_access openid profile
2022-11-15 09:45:39.979 [info] Token available from cache (for scopes email offline_access openid profile), expires in 4206993 milliseconds
2022-11-15 09:45:40.402 [info] Session removed in another window with scopes: email offline_access openid profile
2022-11-15 09:45:40.402 [info] Logging out of session 'f8cdef31-a31e-4b4a-93e4-5f571e91255a/1:live.com:00037FFEA9CBEFA3/ecf74f3f-3d5d-4615-9cae-3d935ef4cc3f'
2022-11-15 09:45:40.402 [info] Sending change event for session that was removed with scopes: email offline_access openid profile
2022-11-15 09:45:40.402 [info] Logged out of session 'f8cdef31-a31e-4b4a-93e4-5f571e91255a/1:live.com:00037FFEA9CBEFA3/ecf74f3f-3d5d-4615-9cae-3d935ef4cc3f' with scopes: email offline_access openid profile
2022-11-15 09:45:40.405 [info] Getting sessions for the following scopes: email offline_access openid profile
2022-11-15 09:45:40.405 [info] Got 0 sessions for scopes: email offline_access openid profile
2022-11-15 09:45:40.501 [error] Error: Unable to transform input.
2022-11-15 09:45:40.563 [error] Error: Unable to parse decrypted password
    at I.$getPassword (vscode-file://vscode-app/c:/Users/${USER}/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1736:49238)
2022-11-15 09:45:40.582 [error] Error: Unable to parse decrypted password
    at I.$getPassword (vscode-file://vscode-app/c:/Users/${USER}/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1736:49238)
2022-11-15 09:45:40.582 [error] Error: Unable to parse decrypted password
    at I.$getPassword (vscode-file://vscode-app/c:/Users/${USER}/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1736:49238)
2022-11-15 09:45:40.582 [error] Error: Unable to parse decrypted password
    at I.$getPassword (vscode-file://vscode-app/c:/Users/${USER}/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1736:49238)
2022-11-15 09:45:40.582 [error] Error: Unable to parse decrypted password
    at I.$getPassword (vscode-file://vscode-app/c:/Users/${USER}/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1736:49238)
2022-11-15 09:45:40.759 [error] Error: Unable to parse decrypted password
    at I.$getPassword (vscode-file://vscode-app/c:/Users/${USER}/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1736:49238)
2022-11-15 09:45:40.760 [error] Error: Unable to parse decrypted password
    at I.$getPassword (vscode-file://vscode-app/c:/Users/${USER}/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1736:49238)
2022-11-15 09:45:40.760 [error] Error: Unable to parse decrypted password
    at I.$getPassword (vscode-file://vscode-app/c:/Users/${USER}/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1736:49238)
2022-11-15 09:45:40.760 [error] Error: Unable to parse decrypted password
    at I.$getPassword (vscode-file://vscode-app/c:/Users/${USER}/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1736:49238)
2022-11-15 09:45:40.760 [error] Error: Unable to parse decrypted password
    at I.$getPassword (vscode-file://vscode-app/c:/Users/${USER}/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1736:49238)
2022-11-15 09:45:40.760 [error] Error: Unable to parse decrypted password
    at I.$getPassword (vscode-file://vscode-app/c:/Users/${USER}/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1736:49238)
2022-11-15 09:45:40.769 [error] Error: Unable to parse decrypted password
    at I.$getPassword (vscode-file://vscode-app/c:/Users/${USER}/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1736:49238)
2022-11-15 09:46:07.788 [info] Getting sessions for all scopes...
2022-11-15 09:46:07.789 [info] Got 0 sessions for all scopes...
2022-11-15 09:46:08.568 [info] Getting sessions for the following scopes: email offline_access openid profile
2022-11-15 09:46:08.568 [info] Got 0 sessions for scopes: email offline_access openid profile
2022-11-15 09:46:09.669 [info] Logging in for the following scopes: email offline_access openid profile
2022-11-15 09:46:12.356 [info] Exchanging login code for token for scopes: email offline_access openid profile
2022-11-15 09:46:12.562 [info] Exchanging login code for token (for scopes: email offline_access openid profile) succeeded!
2022-11-15 09:46:12.563 [info] Setting token for scopes: email offline_access openid profile
2022-11-15 09:46:12.563 [info] Login successful for scopes: email offline_access openid profile
2022-11-15 09:46:12.563 [info] Token available from cache (for scopes email offline_access openid profile), expires in 4284000 milliseconds
2022-11-15 09:46:12.632 [info] Getting sessions for the following scopes: email offline_access openid profile
2022-11-15 09:46:12.632 [info] Got 1 sessions for scopes: email offline_access openid profile
2022-11-15 09:46:12.632 [info] Token available from cache (for scopes email offline_access openid profile), expires in 4283931 milliseconds
2022-11-15 09:46:12.632 [info] Getting sessions for the following scopes: email offline_access openid profile
2022-11-15 09:46:12.632 [info] Got 1 sessions for scopes: email offline_access openid profile
2022-11-15 09:46:12.632 [info] Token available from cache (for scopes email offline_access openid profile), expires in 4283931 milliseconds
2022-11-15 09:46:13.029 [error] Error: Unable to transform input.
2022-11-15 09:46:13.033 [info] Session removed in another window with scopes: email offline_access openid profile
2022-11-15 09:46:13.033 [info] Logging out of session 'f8cdef31-a31e-4b4a-93e4-5f571e91255a/1:live.com:00037FFEA9CBEFA3/0d4cdcdc-e22e-4e45-804a-8459e23c5b60'
2022-11-15 09:46:13.033 [info] Sending change event for session that was removed with scopes: email offline_access openid profile
2022-11-15 09:46:13.033 [info] Logged out of session 'f8cdef31-a31e-4b4a-93e4-5f571e91255a/1:live.com:00037FFEA9CBEFA3/0d4cdcdc-e22e-4e45-804a-8459e23c5b60' with scopes: email offline_access openid profile
2022-11-15 09:46:13.034 [info] Getting sessions for the following scopes: email offline_access openid profile
2022-11-15 09:46:13.034 [info] Got 0 sessions for scopes: email offline_access openid profile
2022-11-15 09:46:13.081 [error] Error: Unable to parse decrypted password
    at I.$getPassword (vscode-file://vscode-app/c:/Users/${USER}/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1736:49238)
```

CmdKey:

```
PS> cmdkey /list | Select-String -Pattern "LegacyGeneric:target=(vscode.+)"

    Target: LegacyGeneric:target=vscodevscode.microsoft-authentication/f8cdef31-a31e-4b4a-93e4-5f571e91255a/1:live.com:00037FFEA9CBEFA3/ecf74f3f-3d5d-4615-9cae-3d935ef4cc3f-3
    Target: LegacyGeneric:target=vscodevscode.microsoft-authentication/microsoft.login.keylist
```

As usual, I have 5 persistent windows open, and signing in from any of them yields the same result.

**Comment by SmithPlatts** [comment]
Me again.

Continued issues with my main work desktop, which I have been waiting for the next release of VSCode to test again.

Updated today:

```
Version: 1.74.0 (user setup)
Commit: 5235c6bb189b60b01b1f49062f4ffa42384f8c91
Date: 2022-12-05T16:38:16.075Z
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.19045
Sandboxed: No
```

Same issue, so I cleared the CmdKey entries, 7, and attempted sign on again:

```
2022-12-08 18:44:29.742 [info] Reading sessions from secret storage...
2022-12-08 18:44:29.743 [info] Got 0 stored sessions
2022-12-08 18:44:32.921 [info] Getting sessions for the following scopes: email offline_access openid profile
2022-12-08 18:44:32.922 [info] Got 0 sessions for scopes: email offline_access openid profile
2022-12-08 18:44:42.175 [info] Getting sessions for all scopes...
2022-12-08 18:44:42.175 [info] Got 0 sessions for all scopes...
2022-12-08 18:44:44.139 [info] Getting sessions for the following scopes: email offline_access openid profile
2022-12-08 18:44:44.141 [info] Got 0 sessions for scopes: email offline_access openid profile
2022-12-08 18:44:45.987 [info] Logging in for the following scopes: email offline_access openid profile
2022-12-08 18:44:52.533 [info] Exchanging login code for token for scopes: email offline_access openid profile
2022-12-08 18:44:52.979 [info] Exchanging login code for token (for scopes: email offline_access openid profile) succeeded!
2022-12-08 18:44:52.979 [info] Setting token for scopes: email offline_access openid profile
2022-12-08 18:44:52.980 [info] Login successful for scopes: email offline_access openid profile
2022-12-08 18:44:52.980 [info] Token available from cache (for scopes email offline_access openid profile), expires in 3830999 milliseconds
2022-12-08 18:44:52.983 [info] Getting sessions for the following scopes: email offline_access openid profile
2022-12-08 18:44:52.983 [info] Got 1 sessions for scopes: email offline_access openid profile
2022-12-08 18:44:52.984 [info] Token available from cache (for scopes email offline_access openid profile), expires in 3830996 milliseconds
2022-12-08 18:44:52.984 [info] Getting sessions for the following scopes: email offline_access openid profile
2022-12-08 18:44:52.985 [info] Got 1 sessions for scopes: email offline_access openid profile
2022-12-08 18:44:52.985 [info] Token available from cache (for scopes email offline_access openid profile), expires in 3830995 milliseconds
2022-12-08 18:44:55.740 [error] Error: Unable to parse decrypted password
    at y.$getPassword (vscode-file://vscode-app/c:/Users/${USER}/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1683:41966)
2022-12-08 18:44:55.748 [info] Session removed in another window with scopes: email offline_access openid profile
2022-12-08 18:44:55.748 [info] Logging out of session 'f8cdef31-a31e-4b4a-93e4-5f571e91255a/1:live.com:00037FFEA9CBEFA3/681b124d-614f-47eb-b91a-95a52784e8a7'
2022-12-08 18:44:55.749 [info] Sending change event for session that was removed with scopes: email offline_access openid profile
2022-12-08 18:44:55.749 [info] Logged out of session 'f8cdef31-a31e-4b4a-93e4-5f571e91255a/1:live.com:00037FFEA9CBEFA3/681b124d-614f-47eb-b91a-95a52784e8a7' with scopes: email offline_access openid profile
2022-12-08 18:44:55.750 [info] Getting sessions for the following scopes: email offline_access openid profile
2022-12-08 18:44:55.750 [info] Got 0 sessions for scopes: email offline_access openid profile
2022-12-08 18:44:56.937 [error] Error: Unable to parse decrypted password
    at y.$getPassword (vscode-file://vscode-app/c:/Users/${USER}/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1683:41966)
```

Again, 5 persistent windows/workspaces open.

Help me Obi Wan Kenobi!

**Comment by TylerLeonhardt** [comment]
@SmithPlatts ah! I totally forgot... I actually need the `Main` Output channel. Can you send me that?

**Comment by SmithPlatts** [comment]
```
2022-12-15 12:08:53.865 [error] TypeError: Cannot read properties of null (reading 'hasNextChunk')
    at p.deletePassword (C:\Users\${USER}\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\code\electron-main\main.js:39:8707)
2022-12-15 12:08:53.866 [error] TypeError: Cannot read properties of null (reading 'hasNextChunk')
    at p.deletePassword (C:\Users\${USER}\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\code\electron-main\main.js:39:8707)
```

**Comment by TylerLeonhardt** [comment]
I made a change ^^  that I think will be helpful here

It's available in the VS Code Insiders Build: https://code.visualstudio.com/Insiders

Please give it a try and let me know how it goes!

**Comment by anarchodin** [comment]
I'm occasionally seeing this error message come up before the one from #130893 does, leading me to suspect that this might be related to something that's retrying and filling the credentials manager. I'm seeing this on 1.75.1, which I _believe_ incorporates the fix mentioned.

**Comment by SmithPlatts** [comment]
I'm really sorry @TylerLeonhardt, I didn't see that you had replied on 17 December!

Unfortunately, on my work machines, I cannot run the Insiders builds. In saying that, I am no longer seeing this, and am on 1.75.1.

**Comment by anarchodin** [comment]
I've now seen this error come up, interspersed with 'decryption failed' messages, apparently filling the output buffer and causing the credential system to be using CPU like nobody's business. This has happened at least twice while I was observing the system, in one of those cases leading to a crash; in the other, deleting all credentials stopped the behaviour.

I have also repeatedly returned to a machine that had VS Code running, which no longer did. I don't know for sure that these crashes are related, but they do leave me with the error from #130893. This only started happening in the past two weeks or so, which makes me suspect it's caused by something that changed in 1.75.

**Comment by TylerLeonhardt** [comment]
I'm going to close this as a duplicate of #130893 and the symptoms seem very similar.

**Comment by VSCodeTriageBot** [comment]
Thanks for creating this issue! We figured it's covering the same as another one we already have. Thus, we closed this one as a duplicate. You can search for [similar existing issues](https://github.com/microsoft/vscode/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aissue+label%3A%22bug%22%20label%3A%22authentication%22). See also our [issue reporting guidelines](https://aka.ms/vscodeissuereporting).

Happy Coding!
